lib/ssh/src/ssh_client_channel.erl

```diff
@@ -60,8 +60,7 @@ The timeout values that can be returned by the callback functions have the same
 semantics as in a `m:gen_server`. If the time-out occurs, `c:handle_msg/2` is called as
 handle_msg(timeout, State).
 """.
--moduledoc(#{since => "OTP 21.0",
```

Comment:
All these functions are callbacks and should be listed as callbacks only as in other parts of OTP such as gen_statem, ssl_cache_api etc. A module may have both callbacks and API functions.

Reply:
Yes I agree. But for ssh it looks funny now.
`Callbacks: Callback Functions`
I would like to changed it into just `Callbacks` (such as in gen_statem for example).

I'm not sure I follow your comment.

See picture below. AFTER on the right

![image](https://github.com/user-attachments/assets/38dd8f4c-39eb-4678-8956-3a8c929d9959)

Reply:
I do no know why it looks like that, I think it should look like in gen_statem. I guess we need to debug the markdown.

Reply:
I don't follow. I think it works that way because extra title is specified "Callback Functions" in title field. removing title field (line below) aligns the output with gen_statem and other modules containing callbacks.